independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanInfo.java

```diff
@@ -63,6 +63,8 @@
 
     private final String name;
 
+    private final boolean removableIfAmbiguous;
```

Comment:
`removableIfAmbiguous` is IMHO confusing.

Reply:
What do you propose?

Reply:
We can't name it `default` unfortunately...

Reply:
I named it `isDefaultBean`. Is that better?